### PR TITLE
Add help messages to the engine parameter schemes

### PR DESCRIPTION
**Pull request by katyafervent**
* Add placeholders to the AKS engine
* Text contaiing help information about some fields
  of cluster and provisioners forms are added.
* Fix typos at GCE cluster creation form

### CODE REVIEW

File: kqueen/engines/manual.py [hunk 1]
```diff
@@ -27,7 +27,8 @@ class ManualEngine(BaseEngine):
                         'apiVersion',
                         'kind'
                     ]
-                }
+                },
+                'help_message': 'Note, that for user description only "name" and "password" is supported'
```
Comment (mzlatkova) [inline kqueen/engines/manual.py]: Only "name" and "password" are supported for the user description.

File: kqueen/engines/jenkins.py [hunk 2]
```diff
@@ -56,7 +56,8 @@ class JenkinsEngine(BaseEngine):
                 'order': 2,
                 'validators': {
                     'required': False
-                }
+                },
+                'help_message': 'Overrides existing parameters of the Jenkins job'
```
Comment (mzlatkova) [inline kqueen/engines/jenkins.py]: the existing

File: kqueen/engines/gce.py [hunk 3]
```diff
@@ -112,7 +115,8 @@ class GceEngine(BaseEngine):
                 'validators': {
                     'required': False,
                     'cidr': True
-                }
+                },
+                'help_message': 'Provide a unique IP range or leave it empty for an auto-assignment'
```
Comment (mzlatkova) [inline kqueen/engines/gce.py]: for auto assignment

File: kqueen/engines/gce.py [hunk 4]
```diff
@@ -61,7 +61,10 @@ class GceEngine(BaseEngine):
                     'required': True,
                     'min': 1,
                     'number': True
-                }
+                },
+                'help_message':
+                    'To enable network policy, your cluster must have at least 2 nodes. '
+                    'The recommended minimum size cluster to run network policy enforcement is 3 Standard-1 instances'
```
Comment (mzlatkova) [inline kqueen/engines/gce.py]: I would rephrase to something like:
To run the network policy enforcement, the cluster must consist of at least 3 ...
Or:
The recommended minimum cluster size to run the network policy enforcement is ...

File: kqueen/engines/manual.py [hunk 5]
```diff
@@ -27,7 +27,8 @@ class ManualEngine(BaseEngine):
                         'apiVersion',
                         'kind'
                     ]
-                }
+                },
+                'help_message': 'Only "name" and "password" are supported for the user description.'
```
Comment (naumvd95) [inline kqueen/engines/manual.py]: username?
